wd_helper.au3

```diff
@@ -680,7 +680,7 @@ EndFunc   ;==>_WD_HighlightElement
 ; Author ........: Danyfirex
 ; Modified ......: mLipok, Danp2
 ; Remarks .......:
-; Related .......: _WD_HighlightElement
+; Related .......: _WD_HighlightElement [Deprecated]
```

Comment:
It is uncommon to place here [Deprecated] .
It is even unusual.

UDF Specs says: [A comma seperated list of functions or structures related to the function.](https://www.autoitscript.com/wiki/UDF-spec#Headers)

Reply:
> It is uncommon to place here [Deprecated] .
It is even unusual.

> UDF Specs says:

- You appear to have issues with my use of "[Deprecated]" in the header. Do you have a suggested alternative?
- The specs are silent AFAICS regarding additional tags like this. Therefore, I would interpret that it is acceptable.
- Who died and named you the spec police? 😜 

Seriously, this `related` entry will be eventually removed when `_WD_HighlightElement` is removed for good so I don't see that it's a big deal. I still need to check the help file functionality with this coding in place, and I may have to make adjustments due to broken links.

Reply:
Ok.
btw.
We just discuss.

Reply:
> btw. We just discuss.

Understood and agree. However, I think it's easy to misinterpret some of your responses. It may be due to the language barrier. It also could just be differences in our communication styles. 🙂

Reply:
Each of them.